apps/cookbook/src/app/showcase/segmented-control-showcase/segmented-control-showcase.component.html

```diff
@@ -2,135 +2,180 @@
+  <code>badge.content</code>
+  or an icon provided as the name of an icon in the
+  <code>badge.icon</code>
+  property.
+  <br />
+  When using an icon the
+  <code>badge.description</code>
+  property should be set which will be used as the
+  <code>aria-label</code>
+  of the badge for accessibility.
+  <br />
+  The background color of the badge can be controlled through
+  <code>badge.themeColor</code>
+  .
+</p>
+
+<cookbook-example-viewer [ts]="badgeExample.codeSnippet">
+  <cookbook-segmented-control-example-with-badge
+    #badgeExample
+  ></cookbook-segmented-control-example-with-badge>
+</cookbook-example-viewer>
+
+<h3>Colors</h3>
+<p>Segmented Control has support for theme colors when used within a Card.</p>
+<p>
+  Simply set the Card's
+  <code>themeColor</code>
+  property and Segmented Control will render in a light or dark variant accordingly.
+</p>
+<cookbook-example-viewer [html]="colorExample.template">
+  <cookbook-segmented-control-example-color
+    #colorExample
+  ></cookbook-segmented-control-example-color>
+</cookbook-example-viewer>
 
-  <h3>Badge</h3>
-  <p>
-    Badges can be applied to Segmented Control in
-    <code>default</code>
-    mode. A badge can be added to any
-    <code>SegmentItem</code>
-    using the
-    <code>badge</code>
-    property.
-  </p>
-  <p>
-    <a
-      href="https://github.com/kirbydesign/designsystem/blob/main/libs/designsystem/src/lib/components/segmented-control/segment-item.ts"
-      class="kirby-external-icon"
-      target="_blank"
-    >
-      See SegmentItem on Github
-    </a>
-  </p>
-  <p>
-    The badge can contain either a plain text string through
-    <code>badge.content</code>
-    or an icon provided as the name of an icon in the
-    <code>badge.icon</code>
-    property.
-    <br />
-    When using an icon the
-    <code>badge.description</code>
-    property should be set which will be used as the
-    <code>aria-label</code>
-    of the badge for accessibility.
-    <br />
-    The background color of the badge can be controlled through
-    <code>badge.themeColor</code>
-    .
-  </p>
+<kirby-divider [hasMargin]="true"></kirby-divider>
 
-  <cookbook-example-viewer [ts]="badgeExample.codeSnippet">
-    <cookbook-segmented-control-example-with-badge
-      #badgeExample
-    ></cookbook-segmented-control-example-with-badge>
-  </cookbook-example-viewer>
+<h2>Accessibility</h2>
+<p>
+  The Segmented Control implements the
+  <a
+    class="kirby-external-icon"
+    target="_blank"
+    href="https://www.w3.org/WAI/ARIA/apg/patterns/tabs/"
+  >
+    Tabs Pattern from ARIA Authoring Practices Guide
+  </a>
+</p>
+<h3>Keyboard support</h3>
+<p>
+  The component has full keyboard support for navigating between and selecting segment buttons .
+  Keyboard navigation will only focus segment buttons, use
+  <kbd>Enter &#8629;</kbd>
+  or
+  <kbd>Space</kbd>
+  to select the element that is currently focused.
+</p>
```

Comment:
Consider simplifying a bit
```suggestion
  Keyboard navigation moves focus between segment buttons, and
  <kbd>Enter &#8629;</kbd>
  or
  <kbd>Space</kbd>
  selects the element that is currently focused.
</p>
```